File: URI_1174.c
#include <stdio.h>
 
int main() {
 
   float arr[100];
        int i,j;
        for (i = 0; i < 100 ; i++) {
            scanf("%f\n",&arr[i] );
        }

        for (j = 0; j < 100; j++) {
            if (arr[j]<=10.0)
            {
                printf("A[%d] = %.1f\n",j,arr[j]);
            }
        }
    return 0;
}

File: URI_1174.cs
using System; 

class URI {

    static void Main(string[] args) { 

        float []arr = new float[100];
        for (int i = 0; i < 100 ; i++) {
            arr[i] = float.Parse(Console.ReadLine());
        }

        for (int i = 0; i < 100; i++) {
            if (arr[i]<=10)
            {
                Console.WriteLine("A["+i+"] = "+arr[i].ToString("F1") );
            }
        }

    }

}
